File: Array Problems - Geeks For Geeks/Level 1/problem10.cpp
#include<iostream>
using namespace std;

void unionOfTwoArray(int arr1[],int arr2[],int size1,int size2){
    int newArr[size1+size2];
    for(int i=0;i<size1+size2;i++){
        for(int j=0;j<size1;j++){
            for(int k=0;k<size2;k++){
                if(arr1[j]==arr2[k]){
                    
                }
            }
        }
    }
}
void intersectionOfTwoArray(int arr1[],int arr2[],int size1,int size2){
    return;
}


int main(){
    cout<<"Enter the size of the first array: ";
    int size1;
    cin>>size1;
    int arr1[size1];
    cout<<"Enter the elements of the array: ";
    for(int i=0;i<size1;i++){
        cin>>arr1[i];
    }

    cout<<"Enter the size of the second array: ";
    int size2;
    cin>>size2;
    int arr2[size2];
    cout<<"Enter the elements of the array: ";
    for(int i=0;i<size2;i++){
        cin>>arr2[i];
    }
    unionOfTwoArray(arr1,arr2,size1,size2);
    intersectionOfTwoArray(arr1,arr2,size1,size2);
    
}   

File: Array Problems - Geeks For Geeks/Level 1/problem10.py

def unionofList(list1,list2):
    list1.extend(list2)
    unionList=set(list1)
    print(unionList)
def intersection(list1,list2):
    size1=len(list1)
    size2=len(list2)
    if(size1<size2):
        size1,size2=size2,size1
    for i in list1:
        if i not in m:
            m[i]=1
        else:
            m[i]+=1
    intersectionList=[]
    for i in list2:
        if i in m and m[i]:
            m[i]-=1
            intersectionList.append(i)
    print(intersectionList)

n=int(input("Enter the size of the first array:"))
list1=[]
print("Enter array elements: ")
while(n>0):
    number=int(input())
    list1.append(number)
    n-=1
m=int(input("Enter the size of the second array:"))
list2=[]
print("Enter array elements: ")
while(m>0):
    number=int(input())
    list2.append(number)
    m-=1
unionofList(list1,list2)
intersection(list1,list2)